course_info/data_migrations/canvas_account_admin_perms.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO lti_permissions_ltipermission (permission, school_id, canvas_role, allow) VALUES
+	('account_admin_tools', '*', 'Account Observer', FALSE),
+	('account_admin_tools', '*', 'AccountAdmin', TRUE),
+	('account_admin_tools', '*', 'Account Admin', TRUE),
+	('account_admin_tools', '*', 'Course Head', FALSE),
+	('account_admin_tools', '*', 'Course Support Staff', FALSE),
+	('account_admin_tools', '*', 'Department Admin', FALSE),
+	('account_admin_tools', '*', 'DesignerEnrollment', FALSE),
+	('account_admin_tools', '*', 'Faculty', FALSE),
+	('account_admin_tools', '*', 'Guest', FALSE),
+	('account_admin_tools', '*', 'Harvard-Viewer', FALSE),
+	('account_admin_tools', '*', 'Help Desk', FALSE),
+	('account_admin_tools', '*', 'Librarian', FALSE),
+	('account_admin_tools', '*', 'ObserverEnrollment', FALSE),
+	('account_admin_tools', '*', 'SchoolLiaison', TRUE),
+	('account_admin_tools', '*', 'Shopper', FALSE),
+	('account_admin_tools', '*', 'StudentEnrollment', FALSE),
+	('account_admin_tools', '*', 'TaEnrollment', FALSE),
+	('account_admin_tools', '*', 'TeacherEnrollment', FALSE),
+	('account_admin_tools', '*', 'Teaching Staff', FALSE);
+
```

Comment:
sql is good. We could check if devops is good with .sql or needs a migrations file for this.